tests/config-ci.js

```diff
@@ -0,0 +1,20 @@
+// this file is for use in CircleCI continuous integration environment
+
+var browserNames = ['chrome', 'firefox', 'internet explorer', 'android'];
```

Comment:
Actually, precedence rules have to be double-checked (see https://github.com/openfisca/legislation-explorer/pull/71#discussion_r108738852).